bitsandbytes/cuda_setup/main.py

```diff
@@ -75,6 +75,8 @@ def generate_instructions(self):
             make_cmd += ' make cuda110'
         elif self.cuda_version_string[:2] == '11' and int(self.cuda_version_string[2]) > 0:
             make_cmd += ' make cuda11x'
+        elif self.cuda_version_string == '121':
```

Comment:
```suggestion
        elif self.cuda_version_string[:2] == '12' and 1 >= int(self.cuda_version_string[2]) >= 0:
```
I suggest this change to support `12.0` either.

Reply:
I think this is the only line necessary for the support of the automatic debugging string. The other parts should work for cuda 12.x.

Reply:
Yes, as long as nothing will be changed from PyTorch or Cuda side it should work for future releases, too.
(I wonder, if this was a question from your point of view?)